<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-0FTMK2BJ" failures="1" tests="4" name="Test" time="24.009" errors="0" timestamp="2020-12-27T15:31:06 IST">
  <testcase classname="Academy.ValidateTitle" name="validateTitle" time="0.064">
    <failure type="java.lang.AssertionError" message="expected [FEATURED COURSES1] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: expected [FEATURED COURSES1] but found [FEATURED COURSES]
at Academy.ValidateTitle.validateTitle(ValidateTitle.java:47)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateTitle -->
  <testcase classname="Academy.ValidateNavigationBar" name="validateNavigationBar" time="0.058"/>
  <testcase classname="Academy.HomePage" name="basePageNavigation" time="1.669"/>
  <testcase classname="Academy.HomePage" name="basePageNavigation" time="1.855"/>
</testsuite> <!-- Test -->
